src/components/mortgage-investment-compare/amortization-table.js

```diff
@@ -12,7 +12,7 @@ const tableDivider = css`
 `
 const fixedTableLayout = css`
   display: table;
-  width: 100%;
+  width: ${window.innerWidth * 0.595}px;
```

Comment:
Related to #58. I had to give the table a fixed width. Let me know how it looks on your end. This should solve the issue with the thead and the tbody being misaligned but it has a secondary minor side effect which I prefer: the scrollbar will be separated from the tbody when you zoom out. We'll come up with a better implementation of this table some time in the future but for now I think this works well

Reply:
I think it looks good:

![image](https://user-images.githubusercontent.com/3756871/66016671-6d311b80-e48c-11e9-86d9-802d58908db8.png)

Reply:
great. Just curious, why isnt' the scrollbar there? is your screen that big? 

![image](https://user-images.githubusercontent.com/9560798/66024456-c3ac5300-e4a8-11e9-94ef-a4fc0795fea0.png)

Reply:
Scrollbars don't show up on MacOS I don't think

Reply:
so do you just scroll without the scrollbar basically?

Reply:
Yep! Using the touchpad. The scroll comes up when I scroll:

![image](https://user-images.githubusercontent.com/3756871/66068253-17975600-e502-11e9-91c9-d971ac4b1ed3.png)